src/fwd.hpp

```diff
@@ -19,6 +19,7 @@
 
 #include <boost/python.hpp>
 #include <Eigen/Core>
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
```

Comment:
After reading the code of numpy v1.11 it seems that this not even needed.
Only NPY_1_7_API_VERSION is required.